api/lang/customResponses.go

```diff
@@ -18,6 +39,26 @@ type ScenarioInfo struct {
 	ExecutionIdentifier string `json:"executionIdentifier"`
 }
 
+type specInfo struct {
+	Heading             string `json:"heading"`
+	ExecutionIdentifier string `json:"executionIdentifier"`	
+}
+
+func getSpecs() (interface{}, error) {
```

Comment:
We already have a specs cache in specInfoProvider.
Other lsp features depend on this cache. 
We can use the cache to get all specs.
Any reason we need to parse the files to get all the specs?